### PR TITLE
feature: Add system notification tray functionality

### PR DESCRIPTION
**Pull request by ehguo**

### CODE REVIEW

**Comment by ehguo** [comment]
Haven't gotten the actual notifications yet, this is just adding the icon plus a context menu.

File: app/main.dev.js [hunk 1]
```diff
@@ -43,6 +44,32 @@ app.on('ready', async () => {
     icon: path.join(__dirname, '../resources/icons/64x64.png')
   });
 
+  mainWindow.on('minimize', function (e) {
+    e.preventDefault();
+    mainWindow.hide();
+  });
+
+  tray = new Tray(path.join(__dirname, '../resources/icons/64x64.png'));
```
Comment (chengsieuly) [inline app/main.dev.js]: For the tray, can you refactor to how `buildMenu` is like?

Comment (chengsieuly) [inline app/main.dev.js]: so a function that `buildTray`

Comment (ehguo) [inline app/main.dev.js]: Should I put this in it's own folder inside electron as well?

Comment (chengsieuly) [inline app/main.dev.js]: Yea, I think that would be a great idea

**Pull request merged**